cs-minibot-platform-src/src/main/java/minibot/BaseHTTPInterface.java

```diff
@@ -119,15 +116,21 @@ public static void main(String[] args) {
 
             // gets (botID, fl, fr, bl, br) from json
             String botName = commandInfo.get("name").getAsString();
-            int fl = commandInfo.get("fl").getAsInt();
-            int fr = commandInfo.get("fr").getAsInt();
-            int bl = commandInfo.get("bl").getAsInt();
-            int br = commandInfo.get("br").getAsInt();
+//            int fl = commandInfo.get("fl").getAsInt();
+//            int fr = commandInfo.get("fr").getAsInt();
+//            int bl = commandInfo.get("bl").getAsInt();
+//            int br = commandInfo.get("br").getAsInt();
+            String fl = commandInfo.get("fl").getAsString();
+            String fr = commandInfo.get("fr").getAsString();
+            String bl = commandInfo.get("bl").getAsString();
+            String br = commandInfo.get("br").getAsString();
 
             // Forward the command to the bot
             Bot myBot = BaseStation.getInstance().getBotManager().getBotByName(botName).get();
-            FourWheelMovement fwmCommandCenter = (FourWheelMovement) myBot.getCommandCenter();
-            return fwmCommandCenter.setWheelPower(fl,fr,bl,br);
+//            FourWheelMovement fwmCommandCenter = (FourWheelMovement) myBot.getCommandCenter();
+//            return fwmCommandCenter.setWheelPower(fl,fr,bl,br);
+            CommandCenter cc =  myBot.getCommandCenter();
+            return cc.sendKV("WHEELS", "<<<<WHEELS," + fl + "," + fr + "," + bl + "," + br + ">>>>");
```

Comment:
sendKV should handle the formation of the delimiter, so value should just be fl + "," + fr + "," + bl + "," + br